canonical-web-and-design/snapcraft.io
# Annotated snapcraft yaml

Fixes #1896

Adds annotated snapcraft.yaml viewer to Python tutorial in first snap flow (Package step).
Allows to download snapcraft.yaml file generated with a snap name chosen by user.

### QA
- ./run or https://snapcraft-io-canonical-web-and-design-pr-1891.run.demo.haus/
- go to Package step of Python first snap flow
- choose name, go to third step (Download your snapcraft.yaml)
- annotated yaml should be shown (with a name chosen before)
- tooltips should be shown when hovering over different sections of the file
- button below it should download the snapcraft.yaml file with a name chosen by user

**Please make sure to read the copy text of "Downloa your snapcraft.yaml" section. My "D" key is misbehaving recently an I may have mae some typos ;)**

<img width="1013" alt="Screenshot 2019-05-09 at 15 56 28" src="https://user-images.githubusercontent.com/83575/57459531-b54c6900-7273-11e9-9f94-af37845bec11.png">
